### ISSUE TITLE
seemingly trivial error not easily resolved

### ISSUE THREAD

**Issue by BenWestbrook** [dotnet/AspNetCore.Docs#17380]
Fresh install Windows 10
Full install Visual Studio 2019 Professional
Create new web application 
Attempt to follow these instructions exactly
Roadblock at following steps:
"
Install-Package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore
Add-Migration CreateIdentitySchema
Update-Database
"
Specifically, the Add-Migration step fails with:

More than one DbContext was found. Specify which one to use. Use the '-Context' parameter for PowerShell commands and the '--context' parameter for dotnet commands.

This scenario has two database instances, MSSQLLocalDB and ProjectsV13

There are no easily found examples of Add-Migration utilizing the -Context option, thus rendering this whole tutorial useless.

How do we translate an SQL instance name to a "DbContext" for the Add-Migration command. An example using the MSSQLLocalDB sure would be helpful.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 804e24dd-7e30-8cf5-2c99-1ffacb4582f6
* Version Independent ID: 3e25547b-e2c9-6634-2c38-2a77207f17d4
* Content: [Scaffold Identity in ASP.NET Core projects](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-3.1&tabs=visual-studio&source=docs#feedback)
* Content Source: [aspnetcore/security/authentication/scaffold-identity.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/scaffold-identity.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
If you follow the directions you end up with one DB context. It's a great error message.

You can

* Delete one context
* Specify the context to use.

**Comment by BenWestbrook** [comment]
Specifically, which step did I miss that caused me to have two DBContext and therefore unable to complete to tutorial?

**Comment by gbjbaanb** [comment]
I did the same, and got the same result. Turned out threre was 2 contexts defined in the project, however as I realised and fixed the offending one (ie renamed it to the correct DBcontext I was working with) I cannot tell you where the problem lies.

I have a feeling it was in the AddEntityFrameworkStores() call, where you will probbaly have cut and pasted the context parameter, but if its not that, there is a bit of user error (admittedly easily done) that's the problem.
